test/fetch_schema.js

```diff
@@ -0,0 +1,2036 @@
+
+function add(feature_test, request_method) {
```

Comment:
@ethernal12 : Daj sheme v neko posebno datoteko (schemas) in notri potem daš za vsak route posebaj file... schema_appointments, schema_messages etc...

Reply:
Schem je kar veliko zato se jih splača dati v vsak svoje file.

Reply:
Misliš folder? Tako da ustvarim folder schemas in potem notri priložim za vsak router posebaj .js file ?

Reply:
@ethernal12 Ja da v folder daš za vsak route svojo shemo da so sheme popolnoma ločene od test fileov... Fetch_schema daj v en direktorij in jo razdeli na več fileov (vsak route posebaj)

Reply:
Ok, bom popravil.